ydb/core/persqueue/partition.h

```diff
@@ -65,8 +66,11 @@ struct TTransaction {
 
     TSimpleSharedPtr<TEvPQ::TEvChangePartitionConfig> ChangeConfig;
     bool SendReply;
-
     TSimpleSharedPtr<TEvPQ::TEvProposePartitionConfig> ProposeConfig;
+
+    //Data Tx
+    THashSet<TString> AffectedSourcesIds;
```

Comment:
Why do we need this? I thought we will use same object for storing changed source Ids and their seq no with usual write.